packages/deeplinks/observability/locators/observability_log_explorer.ts

```diff
@@ -6,9 +6,17 @@
  * Side Public License, v 1.
  */
 
+import { SerializableRecord } from '@kbn/utility-types';
 import { LogExplorerNavigationParams } from './log_explorer';
 
-export type DatasetLocatorParams = LogExplorerNavigationParams;
+// Will become a union once we have more origins
+export interface LocationState extends SerializableRecord {
```

Comment:
**nit:** what if we rename it `ObservabilityLogExplorerLocationState` directly? As far as `LocationState` sounds correct, I would make clear what location state we represent here, in case we implement a similar pattern in future and we define a similar interface.

Reply:
Yeah, I kinda went back and forth on this. `ObservabilityLogExplorerLocationState` was the original name, but then I thought `LocationState` might better alert people to the fact it's the literal `LocationState` tied to history. My thinking was any new locators that make use of it (history location) could also just use `LocationState` and then consumers can instead import that how they wish. 

I'll change it over though, since I've bounced between both and you're effectively giving one a second vote 😁